common/components/TXMetaDataPanel/components/FeeSummary.scss

```diff
@@ -9,3 +9,9 @@
   text-align: center;
   font-size: 14px;
 }
+
+.SchedulingFeeSummary {
+  font-size: 12px;
+  height: auto;
+  min-height: 42px;
+}
```

Comment:
Any reason for the altered styles for scheduled fees?

Reply:
Our formula for calculating total cost is little bit more complicated and includes more variables, thus is bigger than previous one (could take more than one line easily). Example formula display:

`0.0045 ETH + 1 + 4000000000 * 1500000 + 20000000000 * (180000 + 21000) = 0.014504 ETH ~= 0.00 USD`